proposals/records.md

```diff
@@ -83,14 +58,58 @@ A record type contains two synthesized copying members if methods with the same
 signature are not already declared within the record type:
 
 * A protected constructor taking a single argument of the record type.
-* A public parameterless instance method called `Clone` which returns the record type.
+* A public parameter-less virtual instance "clone" method with a compiler-reserved name
 
 The protected constructor is referred to as the "copy constructor" and the synthesized
-body copies the values of all instance fields declared in the input type to the corresponding 
+body copies the values of all instance fields declared in the input type to the corresponding
 fields of `this`.
 
-The `Clone` method returns the result of a call to a constructor with the same signature as the
-copy constructor.
+The "clone" method returns the result of a call to a constructor with the same signature as the
+copy constructor. The return type of the clone method is the containing type, unless a virtual
+clone method is present in the base class. In that case, the return type is the current containing
+type if the "covariant returns" feature is supported and the override return type otherwise. The
+synthesized clone method is an override of the base type clone method if one exists. An error is
+produced if the base type clone method is sealed.
+
+## Positional record members
+
+In addition to the above members, records with a parameter list ("positional records") synthesize
+the following members, if a concrete (i.e. non-abstract) member with the same signature (or name
+if the member is a field or property) is not already present:
```

Comment:
Are explicit fields/properties (inherited or defined on record type) allowed to differ by type, as long as the record parameter type is assignable to the field/property type?